### PR TITLE
Simplify test debug

### PR DESCRIPTION
**Pull request by dgageot**
+ Remove code that's not required
+ Use base images that are already used elsewhere. It might help reduce the number of images we pull

See #2398

### CODE REVIEW

File: integration/debug_test.go [hunk 1]
```diff
@@ -55,23 +53,24 @@ func TestDebug(t *testing.T) {
...
+
 			ns, client, deleteNs := SetupNamespace(t)
 			defer deleteNs()
 
-			stop := skaffold.Debug(test.args...).WithConfig(test.filename).InDir(test.dir).InNs(ns.Name).WithEnv(test.env).RunBackground(t)
+			stop := skaffold.Debug(test.args...).InDir(test.dir).InNs(ns.Name).RunBackground(t)
 			defer stop()
 
 			client.WaitForPodsReady(test.pods...)
-			client.WaitForDeploymentsToStabilize(test.deployments...)
```
Comment (briandealwis) [inline integration/debug_test.go]: The `jib` example is a deployment.

File: integration/debug_test.go [hunk 2]
```diff
@@ -55,23 +53,24 @@ func TestDebug(t *testing.T) {
...
 			for _, depName := range test.deployments {
```
Comment (briandealwis) [inline integration/debug_test.go]: Hmm this should be testing the pods too.  I'll fix that separately.

File: integration/debug_test.go (hunk 1 above)
Comment (dgageot) [reply integration/debug_test.go]: Not sure I understand your point

Comment (briandealwis) [reply integration/debug_test.go]: By deleting this line, the deployments may not be stabilized before the test continues. Right?

Comment (dgageot) [reply integration/debug_test.go]: `GetDeployment` does it

**Comment by dgageot** [comment]
BEFORE:

```
--- PASS: TestDebug/kubectl (212.36s)
--- PASS: TestDebug/kustomize (55.63s)
```

AFTER:

```
--- PASS: TestDebug/kubectl (83.07s)
--- PASS: TestDebug/kustomize (23.91s)
```

**Pull request merged**